deployments/liqo/README.md

```diff
@@ -18,8 +18,10 @@
 | auth.pod.extraArgs | list | `[]` | Extra arguments for the auth pod. |
 | auth.pod.labels | object | `{}` | Labels for the auth pod. |
 | auth.pod.resources | object | `{"limits":{},"requests":{}}` | Resource requests and limits (https://kubernetes.io/docs/user-guide/compute-resources/) for the auth pod. |
+| auth.service.allocateLoadBalancerNodePorts | string | `""` | If service type is LoadBalancer and want to expose the gateway on NodePorts set it to true |
 | auth.service.annotations | object | `{}` | Annotations for the auth service. |
 | auth.service.labels | object | `{}` | Labels for the auth service. |
+| auth.service.loadBalancerIP | string | `""` | If service type is LoadBalancer and you want to use a specific IP address incase of static LB overrite the IP here. |
```

Comment:
```suggestion
| auth.service.loadBalancerIP | string | `""` | Override the IP here if service type is LoadBalancer and you want to use a specific IP address in case of a static LB. |
```